test/Microsoft.AspNetCore.Razor.Test.Common/Langauge/IntegrationTests/IntegrationTestBase.cs

```diff
@@ -184,9 +184,6 @@ protected void AssertDesignTimeDocumentMatchBaseline(RazorCodeDocument document)
             Assert.NotNull(syntaxTree);
             Assert.True(syntaxTree.Options.DesignTimeMode);
```

Comment:
Do we need these other checks here? Why not just pass in the mappings instead of the code document?